traffic/traffic_info.cpp

```diff
@@ -202,7 +202,7 @@ void TrafficInfo::CombineColorings(vector<TrafficInfo::RoadSegmentId> const & ke
   result.clear();
   size_t numKnown = 0;
   size_t numUnknown = 0;
-  size_t numUnexpectedKeys = knownColors.size();
+  [[maybe_unused]] size_t numUnexpectedKeys = knownColors.size();
```

Comment:
?

Reply:
Used only in debug and gcc in release says that it's never used

Reply:
It's better to use `#ifdef DEBUG` in such cases for clarity.

Reply:
Yes, and write ASSERT_GREATER(numUnexpectedKeys--, 0, ());